sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/AiSemanticAttributes.java

```diff
@@ -48,8 +46,8 @@ public final class AiSemanticAttributes {
     public static final AttributeKey<String> OPERATION_NAME =
         stringKey("applicationinsights.internal.operation_name");
 
-    public static final AttributeKey<Long> ITEM_COUNT =
-        longKey("applicationinsights.internal.item_count");
+    public static final AttributeKey<Double> SAMPLE_RATE =
```

Comment:
It would be more specific:

```suggestion
    public static final AttributeKey<Double> INTERNAL_SAMPLE_RATE =
```

Reply:
this whole class is internal, not sure adding INTERNAL_ prefix on every constant is needed